### ISSUE TITLE
Problem with voicevox

### ISSUE THREAD

**Issue by mlg-manxxx** [AlizerUncaged/desktop-waifu#9]
What should I do about this problem?

![image](https://github.com/AlizerUncaged/desktop-waifu/assets/51211531/6060eaf9-c1ac-4c23-9530-6dc50fc37503)

**Comment by Yoishima** [comment]
this also applies to me with elevenlabs, i tried both and same error still

**Comment by Speeder707** [comment]
You check your open AI key? If something happened or it went over on the usage.

**Comment by Yoishima** [comment]
yup, it's valid, and even made a new api key, still the same error showed up

**Comment by wayloft** [comment]
I have this issue as well

**Comment by SoftRuby** [comment]
same here

**Comment by Xenpay** [comment]
same here,,, hahaha

**Comment by Yoishima** [comment]
still not solved huh?
